Makefile

```diff
@@ -28,6 +28,17 @@ GO_TEST_FLAGS=$(shell if [ "$$(uname -m)" = "s390x" ]; then echo "-v"; else echo
 
 .EXPORT_ALL_VARIABLES:
 
+define gofmt-test =
```

Comment:
Handle indentation - For all method body lines add leading tab

Reply:
![image](https://user-images.githubusercontent.com/57443811/75258038-8eedbf00-57ee-11ea-85b4-c849c2cb313b.png)